presto-parquet/src/test/java/io/prestosql/parquet/TestParquetTimestampUtils.java

```diff
@@ -50,7 +53,8 @@ private static void assertTimestampCorrect(String timestampString)
     {
         Timestamp timestamp = Timestamp.valueOf(timestampString);
         Binary timestampBytes = getNanoTime(timestamp, false).toBinary();
-        long decodedTimestampMillis = getTimestampMillis(timestampBytes);
-        assertEquals(decodedTimestampMillis, timestamp.toEpochMilli());
+        DecodedTimestamp decodedTimestamp = decode(timestampBytes);
+        assertEquals(decodedTimestamp.getEpochSeconds() * MILLISECONDS_PER_SECOND + decodedTimestamp.getNanosOfSecond() / NANOSECONDS_PER_MILLISECOND, timestamp.toEpochMilli());
```

Comment:
convert `timestampString` to LDTime so that the assertion does not need multiplcation and division